index.html

```diff
@@ -51,50 +51,57 @@
     <div class="container main-content" >
         <!-- row 1 -->
         <div id="content-search" class="row">
-            <div class="col-sm-4">
-                Text in row 1 column 1
+            <div id="row1-column1" class="col-sm-4">
+                <img id="card-image" src="">
```

Comment:
I was trying some stuff out,   and I think it look better if you went with the "normal" card image size and not the small.  Then create a css class for card-image  and set it's width.

ex.
```
#card-image {
    width: 275px;
}
```

Reply:
ex. https://c1.scryfall.com/file/scryfall-cards/normal/front/4/7/47edde50-81c8-4fe0-ae13-9a10171361bc.jpg?1562273842